WordPress/src/test/java/org/wordpress/android/ui/stats/refresh/utils/StatsDateFormatterTest.kt

```diff
@@ -72,6 +77,42 @@ class StatsDateFormatterTest : BaseUnitTest() {
         assertThat(parsedDate).isEqualTo("Dec 17 - Dec 23")
     }
 
+    @Test
+    fun `prints a week date in string format in granular stats screen`() {
+        whenever(statsTrafficTabFeatureConfig.isEnabled()).thenReturn(true)
```

Comment:
`statsTrafficTabFeatureConfig.isEnabled()` should return false for the granular stats screen.
```suggestion
        whenever(statsTrafficTabFeatureConfig.isEnabled()).thenReturn(false)
```

And `parsedDate` should be "Dec 17 - Dec 18".

Reply:
Hey! Sorry about that @irfano . It wasn't clear because the test description was poorly named. The two tests I added are for testing with the traffic tab feature on. This verifies that when the feature is on, that's when we add the year to the week date picker. I renamed it to be more clear.